### PR TITLE
fix version navigation in release v1.7

### PR DESCRIPTION
**Pull request by harshvkarn**
- In `release v1.7`, navigating selector to version `v1.10` disappears
- modified `_config.yml` in branch `release-1.7`

Signed-off-by: harshvkarn <harshvkarn54@gmail.com>

### CODE REVIEW

**Comment by harshvkarn** [comment]
@zacharysarah 
#9071

**Comment by harshvkarn** [comment]
issue #9356

File: _config.yml [hunk 1]
```diff
@@ -23,11 +23,16 @@ defaults:
       githubbranch: "v1.7.3"
       docsbranch: "release-1.7"
       versions:
+      - fullversion: "v1.10.0"
+          version: "v1.10"
+          githubbranch: "v1.10.0"
+          docsbranch: "release-1.10"
+          url: https://kubernetes.io/
```
Comment (ajaeger) [inline _config.yml]: This should be https://v1-10.docs.kubernetes.io/  - and 1.11 is released and need to be mentioend here.

File: _config.yml [hunk 2]
```diff
@@ -23,11 +23,21 @@ defaults:
       githubbranch: "v1.7.3"
       docsbranch: "release-1.7"
       versions:
+      - fullversion: "v1.11.0"
+          version: "v1.11"
+          githubbranch: "v1.11.0"
+          docsbranch: "release-1.11"
+          url:  https://kubernetes.io/
```
Comment (tengqm) [inline _config.yml]: Please consider use 'https://kubernetes.io/docs' as the URL. Using the "/" URL path makes me nervous.

Comment (harshvkarn) [reply _config.yml]: Done @tengqm

**Comment by tengqm** [comment]
@harshvkarn Looks like fixing the 1.7 branch is not so straightforward. Need to tweak the ci gate.
/lgtm cancel

**Comment by harshvkarn** [comment]
@tengqm so what are the changes you suggest?

**Comment by tengqm** [comment]
@harshvkarn If you take a look at this travis config file here: https://github.com/kubernetes/website/blob/release-1.7/.travis.yml#L15

you can see the CI gate was testing things against the master branch of kubernetes. It will fail for sure since many packages have been added/removed since 1.7 was released.
One thing to try is to add the following commands before line 15:
```
cd $GOPATH/src/k8s.io/kubernetes/
git checkout release-1.7
```

**Comment by harshvkarn** [comment]
@tengqm added the snippets suggested by you to `.travis.yml` file

**Comment by tengqm** [comment]
Finally!
/lgtm

**Pull request merged**